packages/liveblocks-node/src/__tests__/Session.test.ts

```diff
@@ -87,6 +88,14 @@ describe("authorization (new API)", () => {
     ).toThrow("Invalid room name or pattern");
   });
 
+  test("throws when room or pattern is undefined", () => {
+    const session = makeSession();
+    expect(() =>
+      // @ts-expect-error - Deliberate incorrect undefined value
+      session.allow(undefined, session.FULL_ACCESS).serializePermissions()
```

Comment:
This will also fail if it's `null`, `false`, a number, etc etc. Maybe change the runtime check to `typeof roomIdOrPattern === 'string'` instead?